spec/spec_helper.rb

```diff
@@ -1,55 +1,12 @@
-require 'simplecov'
-SimpleCov.start 'rails'
+# frozen_string_literal: true
 
 ENV["RAILS_ENV"] = "test"
 
 require File.expand_path("../dummy/config/environment.rb",  __FILE__)
 
-require 'ffaker'
-require 'rspec/rails'
-require 'database_cleaner'
-require 'capybara/rspec'
-require 'capybara/rails'
-require 'capybara/poltergeist'
-Capybara.register_driver(:poltergeist) do |app|
-  Capybara::Poltergeist::Driver.new app, timeout: 90
-end
-Capybara.javascript_driver = :poltergeist
-Capybara.default_max_wait_time = 10
+require "rspec/rails"
 
-Dir[Rails.root.join("spec/support/**/*.rb")].each {|f| require f}
+Dir[File.join(File.dirname(__FILE__), "/support/**/*.rb")].each { |file| require file }
 
-require 'spree/testing_support/factories'
-require 'spree/testing_support/url_helpers'
-require 'spree/testing_support/controller_requests'
-require 'spree/testing_support/authorization_helpers'
-require 'spree/testing_support/capybara_ext'
-
-FactoryBot.find_definitions
-
-RSpec.configure do |config|
-  config.mock_with :rspec
-  config.use_transactional_fixtures = false
-
-  config.include Spree::TestingSupport::ControllerRequests, type: :controller
-  config.include Spree::TestingSupport::UrlHelpers
-  config.include FactoryBot::Syntax::Methods
-
-  config.extend Spree::TestingSupport::AuthorizationHelpers::Request, type: :feature
-
-  config.before(:suite) do
-    DatabaseCleaner.strategy = :transaction
-    DatabaseCleaner.clean_with(:truncation)
-  end
-
-  config.before(:each) do
-    DatabaseCleaner.strategy = RSpec.current_example.metadata[:js] ? :truncation : :transaction
-    DatabaseCleaner.start
-  end
-
-  config.after(:each) do
-    DatabaseCleaner.clean
-  end
-
-  Capybara.javascript_driver = :poltergeist
-end
+require "spree_static_content/factories"
+require "solidus_support/extension/spec_helper"
```

Comment:
we should add the `solidus_support/extension/feature_helper` here instead, and it will already contain [`rails_helper`](https://github.com/solidusio/solidus_support/blob/c552a2aa80f60c4964857d9b6401aede84e6ee6a/lib/solidus_support/extension/rails_helper.rb) and [`spec_helper`](https://github.com/solidusio/solidus_support/blob/c552a2aa80f60c4964857d9b6401aede84e6ee6a/lib/solidus_support/extension/spec_helper.rb) from `solidus_support`. They will already provide almost all the other code that you moved into support files (databse cleaner, capybara configuration, factories) so we can probably remove those files.